parachain/pallets/basic-channel/rpc/Cargo.toml

```diff
@@ -0,0 +1,52 @@
+[package]
+name = "snowbridge-basic-channel-rpc"
```

Comment:
Alll new crates should be added to the cargo workspace in the toplevel Cargo.toml. See a guide for that here: https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html